tests/test_urban_nature_access.py

```diff
@@ -0,0 +1,134 @@
+# coding=UTF-8
+"""Tests for the Urban Nature Access Model."""
+import unittest
+import tempfile
+import shutil
+import os
+import random
+
+import pygeoprocessing
+import numpy
+from osgeo import gdal
+from osgeo import osr
```

Comment:
Order alphabetically?

Reply:
Implemented in one of my recent commits via `isort`!